metadata.json

```diff
@@ -10,23 +10,23 @@
   "dependencies": [
     {
       "name": "garethr/docker",
-      "version_requirement": ">= 3.5.0 <6.0.0"
+      "version_requirement": ">= 5.0.0 < 6.0.0"
```

Comment:
Changelog mentioned something fixed with Puppet 4.3, hence the large version bump